setup.cfg

```diff
@@ -50,6 +50,8 @@ install_requires =
     bokeh~=3.3.2
     holoviews
     dash~=2.11.1
+    # Required for Vector database:
```

Comment:
For conciseness, I prefer to have in-line comments only to resolve potentially misleading lines. This one seems pretty clear, no need to explain.
```suggestion
```